CMakeLists.txt

```diff
@@ -201,9 +201,9 @@ elseif(GNU_GCC OR LLVM_CLANG)
         # TODO(rryan): macOS can use SSE3, and possibly SSE 4.1 once
         # we require macOS 10.12.
         # https://stackoverflow.com/questions/45917280/mac-osx-minumum-support-sse-version
-      elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "arm")
+      elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(aarch64|arm).*")
         add_compile_options(
-          -mflat-abi=hard
```

Comment:
Didn't some users build Mixxx on an RPi? How did that ever work? 😕

Reply:
Has never been used, obviously.